libr/debug/p/native/xnu/xnu_debug.c

```diff
@@ -516,53 +516,12 @@ int xnu_get_vmmap_entries_for_pid (pid_t pid) {
 	return n;
 }
 
-static int get_bits () {
```

Comment:
try running sys/ios-cydia.sh


> On 26 Apr 2016, at 15:41, Riccardo Schirone <notifications@github.com> wrote:
> 
> In libr/debug/p/native/xnu/xnu_debug.c <https://github.com/radare/radare2/pull/4717#discussion_r61086646>:
> 
> > @@ -516,53 +516,12 @@ int xnu_get_vmmap_entries_for_pid (pid_t pid) {
> >  	return n;
> >  }
> >  
> > -static int get_bits () {
> git grep SAME_BITNESS . doesn't return anything except this line in radare2 source code. And since it is defined in a .c file I think it's used(and should be used) only there. Is it used in other repositories?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/radare/radare2/pull/4717/files/9c8878fdc8e00493672812035db5c3efdcd50835#r61086646>